src/data_structure/MGA_data.jl

```diff
@@ -0,0 +1,209 @@
+#############################################################################

```

Comment:
Generally speaking, all our filenames are in lower case - this filename seems to be departing from the convention

Reply:
Thanks, yes that makes sense